### PR TITLE
Initial refactor of web_scraper - Includes config class

### PR DESCRIPTION
**Pull request by matthboers**
Most parameters that were passed individually are now stored in the `WebConfig` class. This class includes handy methods to quickly generate a configuration to work with, for the tests. This should hopefully make it easier to import a configuration file in the future. 

The other change is removing the url from `scrape_medicine_page` and replacing it with eu number, as the URL is not used anymore and it only adds an unnecessary step in the process.

### CODE REVIEW

**Review by elioverhoef — APPROVED** [review]
Goed! Dubbele import in web_scraper_test is gefixt

File: scraping/utilities/web/config_objects.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+                Download all the saved links. This has no effect when one of the previous tasks has not been run.
+            download_refused (bool):
+                Refused files are an edge case, and can slow us down. They are not parsed by the PDF module.
+            download_annex10 (bool):
+                Annex10 is
+            filtering (bool):
+                Retry failed downloads. This has no effect is download has not been run.
+
+        Returns:
+            Modified config class
```
Comment (Koenvisser) [inline scraping/utilities/web/config_objects.py:79]: mist return type

File: scraping/utilities/web/config_objects.py [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+    def set_parallel(self, enabled: bool = True):
+        """
+        Sets the parallelized flag to true. Simply here to allow Config class oneliners.
+
+        Args:
+            enabled (bool): Boolean determining whether scraper runs tasks in parallel
+
+        Returns:
+            Modified Config class
```
Comment (Koenvisser) [inline scraping/utilities/web/config_objects.py:98]: mist return type

File: scraping/utilities/web/config_objects.py [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+        """
+        Instead of having the code gather all medicines from the EC website itself,
+        we can supply a list of medicines for the code to use.
+        Comes in handy if you want to test a smaller set of medicines.
+
+        Args:
+            medicine_list (list[(str, str, int, str)]): A list of medicines, structured as
+                `{url, eu_num_full, medicine_type, eu_num_short}`
+
+        Returns: Modified Config class
```
Comment (Koenvisser) [inline scraping/utilities/web/config_objects.py:114]: mist return type

File: scraping/web_scraper/download.py [hunk 4]
```diff
@@ -261,7 +260,7 @@ def download_annex10_files(data_filepath: str, urls_dict: json_helper.JsonHelper
     Returns:
         None: This function returns nothing.
```
Comment (Koenvisser) [inline scraping/web_scraper/download.py:261]: Returns None is niet nodig

File: scraping/web_scraper/download.py [hunk 5]
```diff
@@ -261,7 +260,7 @@ def download_annex10_files(data_filepath: str, urls_dict: json_helper.JsonHelper
...
     """
-    target_path = data_filepath + "/json"
+    target_path = data_filepath + "/annex_10"
```
Comment (Koenvisser) [inline scraping/web_scraper/download.py:263]: f string is iets mooier

**Review by TimKragten04 — APPROVED** [review]
Duimpie

**Review by Koenvisser — APPROVED** [review]

**Pull request merged**